src/loaders/compute_tools/tool_result_parser.py

```diff
@@ -0,0 +1,105 @@
+import copy
+import os
+from pathlib import Path
+from typing import Dict, List, Set
+
+import jsonlines
+import pandas as pd
+
+from src.loaders.compute_tools.tool_common import GenomeTuple
+
+# used as collection_and_field_names.FLD_KBASE_ID
+TOOL_KBASE_ID = "kbase_id"
```

Comment:
Shouldn't this be imported from `collection_and_field_names`?

Reply:
I need the logic compatible with python 3.8. I cannot import collection_and_field_names

Reply:
Oh right, crap. Ok, maybe we should separate the main file from the collection annotations. If you want to do that here that's fine, otherwise could you make an issue

Reply:
👍 add another AC in the ticcket. https://kbase-jira.atlassian.net/browse/RE2022-218

Reply:
:+1:

Reply:
We need to be a bit careful about this though - the reason I put it all in one place was so the CLI could pull the collection information easily from the file. If we split the collection names from the annotations we might forget to add the annotations